tests/ImageProcessorCore.Tests/Processors/ProcessorTestBase.cs

```diff
@@ -23,6 +23,7 @@ public abstract class ProcessorTestBase
         {
             //"TestImages/Formats/Jpg/Floorplan.jpeg", // Perf: Enable for local testing only
             "TestImages/Formats/Jpg/Calliphora.jpg",
+            "TestImages/Formats/Jpg/IMG_5084.JPG",
```

Comment:
Yes, they are pictures from my last family vacation... All I was after was the meta data and didn't think it appropriate to put them out there in the original form. Will have to look around for some good test images.